examples/tutorials/data/cta.py

```diff
@@ -355,28 +350,29 @@
 
 
 ######################################################################
-# CTA performance files
-# ---------------------
+# Latest CTAO performance files
```

Comment:
Why not stating the config directly as the performances files might change but not the docs once published.

Reply:
Is it better, now?